en/docs/deploy-and-publish/deploy-on-gateway/deploy-api/revision-deployment-workflow.md

```diff
@@ -0,0 +1,78 @@
+# Revision Deployment Workflow
+
+In this section let's see how to add a simple approval workflow to control the deployment of revisions in WSO2 API Manager. When the revisions deployment workflow is activated, a gateway can simultaneously have both a deployed revision and a pending revision. In this setup, developers can submit a revision deployment request to their chosen gateway.
```

Comment:
Shall we imply that the deployment is managed with the approval of the administrator?